maven-archetype-plugin/pom.xml

```diff
@@ -210,13 +210,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
```

Comment:
There is no report in `maven-plugin-plugi`

Reply:
What I shoud add?

Reply:
`maven-plugin-report-plugin`